src/main/java/site/model/Speaker.java

```diff
@@ -13,31 +16,31 @@
 
 import site.config.Globals;
 
-import java.util.HashSet;
-import java.util.Set;
-
 @Entity
 @Inheritance(strategy = InheritanceType.SINGLE_TABLE)
 public class Speaker extends User {
 
     /**
      *
      */
+	
     private static final long serialVersionUID = 1L;
 
     @Column(length = 1024)
     private String bio;
 
     private String headline;
 
-    private String twitter;
+    private String twitter;	
 
     private Boolean featured = false;
 
     private Boolean accepted = false;
 
     @Lob
     private byte[] picture;
+    
+    private String videos;
```

Comment:
I'm wondering why did you prefer to have them as comma separated values, instead of a List<String> coming from a 1 to many relation?

Reply:
* We choose a String field as the forms when working with speaker would become too complicated 
 * The admin panel for add/edit a speaker would be inconvenient

Reply:
@I-Mircheva can you make videos to be "videoLink" and to be just one URL mapped with 

@Column(length = 500)
private String videoLink;

for example so we dont need a migrate sql.